code/_compile_options.dm

```diff
@@ -137,7 +137,7 @@
 #define CBT
 #endif
 
-#if !defined(CBT) && !defined(SPACEMAN_DMM)
+#if !defined(CBT) && !defined(SPACEMAN_DMM) && !defined(OPENDREAM)
```

Comment:
An important consideration here is that people using OD still need to build TGUI manually, so you may want to add a slightly different message if opendream is defined.